source/install/install-ubuntu-2204-postgresql.rst

```diff
@@ -0,0 +1,109 @@
+.. _install-ubuntu-2204-postgresql:
+
+Installing PostgreSQL Database Server
+-------------------------------------
+
+Install and set up the database for use by the Mattermost server. You can install either PostgreSQL or MySQL.
+
+Assume that the IP address of this server is 10.10.10.1.
+
+**To install PostgreSQL on Ubuntu Server 22.04:**
+
+1. Log in to the server that will host the database and issue the following command:
+
+  ``sudo apt install postgresql postgresql-contrib``
+
+  When the installation is complete, the PostgreSQL server is running, and a Linux user account called *postgres* has been created.
+  
+By default on Ubuntu 22.04 this should be Postgres 14, however, Confirm this version is PostgreSQL version 10 or higher. You can do this by running ``psql -V``. If this isn't version 10 or higher you can reference the PostgreSQL documentation `here <https://www.postgresql.org/download/linux/ubuntu/>`__ to install an updated version of PostgreSQL.
```

Comment:
We may be able to take out this line all together. Ubuntu 22.04 defaults to Postgres 14+, so there shouldn't be a need to validate the version anymore.

Reply:
I think we should probably just take it out altogether.

Reply:
I figure by the time we release this they'll have added the docs for it. What do you think?